### PR TITLE
[FIX] account: aged partner report in multi-company

### LINKED ISSUE #23072: [BUG] Aged partner report in multi-company
Ready? Go!

**This:** https://github.com/odoo/enterprise/blob/6fb0346735c56a093c749a8ba898757a3eece274/account_reports/models/account_aged_partner_balance.py#L36
**does not parse the selected companies filter, so that (eg):**

https://github.com/odoo/odoo/blob/b4bda28f0daf6188a8575877ea9a2f68497686b7/addons/account/report/account_aged_partner_balance.py#L30
**cannot be construed with the selected companies, which for example here:**
https://github.com/odoo/odoo/blob/b4bda28f0daf6188a8575877ea9a2f68497686b7/addons/account/report/account_aged_partner_balance.py#L54

**causes the ignorance of the selected filter.**

------------------------------

Out from the current demo.odoo.com, this filter is well intended to exist:
http://nimb.ws/RGYdML

-------------------------------------
**Observed behaviour:**

- The company filter is just plainly ignored (always takes current company of the user)

**Desired behaviour:**
- The company filter is respected so that if a due account on the child company has been paid on the parent company it would not show.

- This has another implication to look for: Even though I only have access to the child company, any account paid & reconciled on the parent company ought not to show as an open balance.

### PR DESCRIPTION
**Pull request by blaggacao**
Closes #23072

Please see commit message and issue report for reference

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by blaggacao** [comment]
Test results unrelated...

**Comment by Yenthe666** [comment]
@qdp-odoo could you have a look whenever you have time?

File: addons/account/report/account_aged_partner_balance.py [hunk 1]
```diff
@@ -158,6 +158,8 @@ def _get_partner_move_lines(self, account_type, date_from, target_move, period_l
         for partner in partners:
             if partner['partner_id'] is None:
                 partner['partner_id'] = False
+            if partner_id == 1023:
+                import wdb; wdb.set_trace()  # breakpoint ee8a7da3 //
```
Comment (qdp-odoo) [inline addons/account/report/account_aged_partner_balance.py]: oops

Comment (blaggacao) [reply addons/account/report/account_aged_partner_balance.py]: Life saver!

**Comment by blaggacao** [comment]
I think that's it, thanks! @qdp-odoo

**Comment by qdp-odoo** [comment]
fix landed in saas-14 at revision https://github.com/odoo/odoo/commit/fcaf459a89b721e2ebc8b758980c76f27052e24f

thanks for the contribution :-)

**Pull request closed without merging**